website/docs/wallet/withdraw-validator.md

```diff
@@ -108,23 +91,23 @@ In order to withdraw, you should have the following items ready:
 }>
 <TabItem value="partial">
 
-## Step-by-Step Guide: Partial Withdrawals (Earnings Only)
+## Option 1: Partial (earnings) withdrawals
 
 :::caution
-⚠️ **DO NOT ATTEMPT TO CHANGE YOUR BLS KEYS ON MAINNET** until the client software is ready to accept your BLS key changes. This is feature is not yet available on mainnet at the time of writing on February 1st, 2023
+**DO NOT ATTEMPT TO CHANGE YOUR BLS KEYS ON MAINNET** until the client software is ready to accept your BLS key changes. This is feature is not yet available on mainnet at the time of writing on February 1st, 2023
```

Comment:
Removing emojis to maintain consistency with the way we use admonitions throughout the corpus.